### PR TITLE
Detonation of items by projectiles is now logged

### PR DESCRIPTION
**Pull request by optimumtact**

### CODE REVIEW

File: code/game/objects/items/flamethrower.dm [hunk 1]
```diff
@@ -239,6 +239,7 @@
 	if(damage && attack_type == PROJECTILE_ATTACK && P.damage_type != STAMINA && prob(15))
 		owner.visible_message("<span class='danger'>\The [attack_text] hits the fueltank on [owner]'s [name], rupturing it! What a shot!</span>")
 		var/target_turf = get_turf(owner)
+		log_game("A fueltank held by [owner] was detonated by a proectile hit")
```
Comment (nemvar) [inline code/game/objects/items/flamethrower.dm]: proectile

Comment (Fhaxaris) [reply code/game/objects/items/flamethrower.dm]: proyectile

Comment (nemvar) [reply code/game/objects/items/flamethrower.dm]: The logging also seems a bit odd. Is there really anything wrong with being more specific here?
"The fuel tank of [name] detonated from getting hit by [hitby]. Holder: [owner] ([owner.real_name])"

File: code/game/objects/items/grenades/grenade.dm [hunk 2]
```diff
@@ -122,6 +122,8 @@
 	var/obj/item/projectile/P = hitby
 	if(damage && attack_type == PROJECTILE_ATTACK && P.damage_type != STAMINA && prob(15))
 		owner.visible_message("<span class='danger'>[attack_text] hits [owner]'s [src], setting it off! What a shot!</span>")
+		log_game("A grenade was detonated by a laser")
```
Comment (nemvar) [inline code/game/objects/items/grenades/grenade.dm]: Any kind of non stamina projectile that deals damage can detonate grenades, not just lasers.

File: code/game/objects/items/flamethrower.dm (hunk 1 above)
Comment (optimumtact) [reply code/game/objects/items/flamethrower.dm]: im lazy

**Comment by optimumtact** [comment]
thanks spacedad

**Pull request merged**